src/NuGet.Core/NuGet.Packaging/ContentModel/ContentPropertyDefinition.cs

```diff
@@ -144,6 +142,21 @@ public virtual bool TryLookup(string name, PatternTable table, out object value)
             return false;
         }
 
+        private static bool ContainsSlash(string name)
```

Comment:
Did we get a bug on this? If not, I'm going to file one - I'm seeing the same thing.